docs/architecture/microservices/multi-container-microservice-net-applications/subscribe-events.md

```diff
@@ -100,7 +100,7 @@ As mentioned earlier in the architecture section, you can have several approache
 
 - Using [transaction log mining](https://www.scoop.it/t/sql-server-transaction-log-mining).
 
-- Using the [Outbox pattern](http://gistlabs.com/2014/05/the-outbox/). This is a transactional table to store the integration events (extending the local transaction).
+- Using the [Outbox pattern](https://gistlabs.wordpress.com/2014/05/the-outbox/). This is a transactional table to store the integration events (extending the local transaction).
```

Comment:
@mairaw this link doesn't exist anymore. What can we do?

Reply:
PR #15213 used http://www.kamilgrzybek.com/design/the-outbox-pattern/ to fix the same link on a different article

Reply:
just updated the link!